gremlin-events/README.md

```diff
@@ -0,0 +1,50 @@
+## Overview
+
+View, rerun, and halt Gremlin attacks directly from Datadog!
+
+Pairing Gremlin with Datadog's [Events][1] is an effective way to add context failure testing context to your Datadog workflows. 
```

Comment:
> is an effective way to add context failure testing context

is an effective way to add failure testing context

Reply:
Fixed, Thanks!